python.jsf

```diff
@@ -27,12 +27,16 @@
 	"\""		string_dq_1	recolor=-1
 	"a-zA-Z_"	ident		noeat
 	"{}"		brace		recolor=-1
-	"@"		decorator	recolor=-1
+	"@"		maybe_decorator
 
 :brace Brace
 	*		idle		noeat
 
 # annotations
+:maybe_decorator Idle
+	"a-zA-Z_"	decorator	recolor=-2
+	" (\t\r\n"	idle		noeat
```

Comment:
Since this is just maybe a decorate I would use `* idle noeat` here as a catch all since we do not really know (and should not limit) which char comes next.

Reply:
Thanks for the feedback! The `* idle noeat` is definitely the way to go.